frontend/device/kobo/powerd.lua

```diff
@@ -128,6 +128,14 @@ function KoboPowerD:init()
             -- Does this device's NaturalLight use a custom scale?
             self.fl_warmth_min = self.device.frontlight_settings["nl_min"] or self.fl_warmth_min
             self.fl_warmth_max = self.device.frontlight_settings["nl_max"] or self.fl_warmth_max
+            -- If this device has a mixer, we can use the ioctl for brightness control, as it's much lower latency.
+            if self.device:hasNaturalLightMixer() then
+                local kobolight = require("ffi/kobolight")
+                local ok, light = pcall(kobolight.open)
+                if ok then
+                    self.device.frontlight_settings["frontlight_ioctl"] = light
```

Comment:
Or just a dot?

Reply:
@Frenzie: I'm not sure I follow ;).

Reply:
As in self.device.frontlight_settings.frontlight_ioctl is the same thing.